### DISCUSSION TITLE
LDAP without BIND_DN or BIND_PASSWORD

### DISCUSSION THREAD

**Discussion by NilsBaumgartner1994** [directus/directus#9429] [Feature Requests]
Services like GitLab allow LDAP configuration without BIND_DN or PASSWORD:
This would be a nice feature, which my students could use then.

```
 label: 'LDAP'
    host: 'ldap.uni-osnabrueck.de'
    port: 636
    uid: 'uid'
    method: 'ssl' # "tls" or "ssl" or "plain"
#     bind_dn: 'ou=people,dc=uni-osnabrueck,dc=de'
#     password: '_the_password_of_the_bind_user'
    active_directory: false
    allow_username_or_email_login: false
    block_auto_created_users: true
    base: 'dc=uni-osnabrueck,dc=de'
```

**Comment by aidenfoxx** [comment]
@NilsBaumgartner1994 It feels like a feature that's going to be uncommon, but you're free to submit a pull request! 😄

EDIT: The reason this isn't included by default is because you'd lose some features when not configuring a bind user, such as authentication by attribute, and verifying session validity on auth refresh. So that might be confusing for some people.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
